plugins/modules/create_blueprint.py

```diff
@@ -119,10 +129,21 @@ def main():
     toml_file = (
         f'name = "{module.params["name"]}"\n'
         f'description = "{description}"\n'
-        f'version = "{module.params["version"]}"\n'
-        f"\n"
     )
 
+    blueprint_version = '0.0.1'
+    blueprint_exists = True
+    results = weldr.api.get_blueprints_info(module.params['name'])
```

Comment:
wrap this in a try/except block to fail gracefully with `module.fail_json()` and a relevant error message if this run against a host where the service isn't started or is otherwise not available